src/commands/api-mesh/cache/purge.js

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const { Command } = require('@oclif/command');
+
+const logger = require('../../../classes/logger');
+const { initSdk, initRequestId, promptConfirm } = require('../../../helpers');
+const {
+	ignoreCacheFlag,
+	autoConfirmActionFlag,
+	cachePurgeAllActionFlag,
+} = require('../../../utils');
+const { getMeshId, cachePurge } = require('../../../lib/devConsole');
+const chalk = require('chalk');
+
+require('dotenv').config();
+
+class CachePurgeCommand extends Command {
+	static flags = {
+		ignoreCache: ignoreCacheFlag,
+		autoConfirmAction: autoConfirmActionFlag,
+		purgeAllAction: cachePurgeAllActionFlag,
```

Comment:
The custom help label does not change the underlying command. I think you will need to change the name of the flag itself:

```suggestion
		all: cachePurgeAllActionFlag,
```

Reply:
Done, have implemented the change, please have a look